qualtran/bloqs/basic_gates/y_gate.py

```diff
@@ -65,3 +102,86 @@ def as_cirq_op(
 
     def _t_complexity_(self) -> 'TComplexity':
         return TComplexity(clifford=1)
+
+    def wire_symbol(
+        self, reg: Optional['Register'], idx: Tuple[int, ...] = tuple()
+    ) -> 'WireSymbol':
+        if reg is None:
+            return Text('')
+        return TextBox('Y')
+
+
+@bloq_example
+def _y_gate() -> YGate:
+    y_gate = YGate()
+    return y_gate
+
+
+_Y_GATE_DOC = BloqDocSpec(bloq_cls=YGate, examples=[_y_gate], call_graph_example=None)
+
+
+@frozen
+class CYGate(Bloq):
+    """A controlled Y gate.
+
+    Registers:
+        ctrl: The control qubit.
+        q: The target qubit.
+    """
+
+    @cached_property
+    def signature(self) -> 'Signature':
+        return Signature.build(ctrl=1, q=1)
```

Comment:
Should we use `ctrl` and `target` instead of `ctrl` and `q` ? That's what the `CNOT` gate uses.